MojioSDK/Models/Vehicles/TripProperties.swift

```diff
@@ -95,3 +95,32 @@ public struct TripProperties: TripPropertiesModel {
         try container.encodeIfPresent(self.details, forKey: .details)
     }
 }
+
+public struct TripPropertiesUpdate: Codable {
+    public let vehicleId: String
+    public let purpose: String?
+    public let notes: String?
+    
+    public enum CodingKeys: String, CodingKey {
+        case vehicleId = "vehicleid"
```

Comment:
@andriy-morgunov-epam camel case is more usual for coding keys, so consider "vehicle_id"

Reply:
@stepanrazin Not sure I've got your comment. The API contract is below. 

> curl --location --request PATCH 'http://stg1-sfapi.moj.io:44202/trips/TRIP-GUID/details' \
--header 'Content-Type: application/json' \
--header 'Authorization: Bearer TOKEN-HERE' \
--data-raw '{
"vehicleid":"VEHICLE-GUID",
"purpose":"test purpose",
"notes":"test notes"
}'

https://1mojio.atlassian.net/wiki/spaces/API/pages/789020865/TDG+eLogBook+API+Changes